apps/parsley/src/context/LogContext/index.tsx

```diff
@@ -27,8 +27,11 @@ import {
 } from "constants/enums";
 import { QueryParams } from "constants/queryParams";
 import { useFilterParam } from "hooks/useFilterParam";
+import { useParsleySettings } from "hooks/useParsleySettings";
 import { useQueryParam } from "hooks/useQueryParam";
+import { useSections } from "hooks/useSections";
```

Comment:
If you convert `sectionsEnabled` to `sectioningEnabled`, this should probably become `useSectioning`